File: LLM-Translations/GPT-4/codenet/Python/C/s142939923.c
#include <stdio.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);

    // A + B
    int x = a + b;
    // A - B
    int y = a - b;
    // A * B
    int z = a * b;

    int max = x;
    if (y > max) {
        max = y;
    }
    if (z > max) {
        max = z;
    }
    printf("%d\n", max);

    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s142939923.go
package main

import (
	"fmt"
)

func main() {
	var a, b int
	fmt.Scan(&a, &b)

	// A + B
	x := []int{a + b}
	// A - B
	x = append(x, a-b)
	// A * B
	x = append(x, a*b)

	max := x[0]
	for _, value := range x {
		if value > max {
			max = value
		}
	}

	fmt.Println(max)
}

// 
